src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -105,6 +105,7 @@ export default class TutorialView {
         this.$tutorialToggle = $(SELECTORS.DOM_SELECTORS.TOGGLE_TUTORIAL);
         this.$tutorialPrevious = this.$tutorialView.find(SELECTORS.DOM_SELECTORS.PREV);
         this.$tutorialNext = this.$tutorialView.find(SELECTORS.DOM_SELECTORS.NEXT);
+        this._onAirportChangeHandler = this.onAirportChange.bind(this);
```

Comment:
you'll want to add a `._setupHandlers()` method that is called after `._init()` in the constructor.  Then move this assignment there.

Reply:
done